server/chat-plugins/info.js

```diff
@@ -2292,7 +2292,7 @@ const commands = {
 		if (!target) return this.parse('/help htmlbox');
 		target = this.canHTML(target);
 		if (!target) return;
-
+		if (target.includes('\n')) target = Chat.removeLineBreaksInHTML(target);
```

Comment:
The `if` part is unnecessary.

Reply:
`removeLineBreaksInHTML` seems kind of verbose, but I can't think of a better name.

`collapseNewlinesInHTML`?

Reply:
`collapseLineBreaksHTML`?

Reply:
I mean, those are just as verbose, maybe `collpaseHTML`?

Reply:
Yeah, but they're easier to understand.

Reply:
I currently like `collapseLineBreaksHTML` the most.

Reply:
I have no strong opinion on it, so sure.